Gemfile

```diff
@@ -53,4 +55,5 @@ end
 
 group :staging, :production do
   gem "rails_stdout_logging"
+  gem "rails_12factor"
```

Comment:
Have a look at https://github.com/heroku/rails_12factor -- I think it includes the `rails_stdout_logging` so slightly redundant to include both.